cubedash/summary/_stores.py

```diff
@@ -1561,6 +1563,21 @@ def get_dataset_footprint_region(self, dataset_id):
             row.region_code,
         )
 
+    def get_dataset_uris(self, dataset: Dataset):
+        """
+        Get a representation of the data URIs so that they
+        can be linked to to download them.
+        """
+        dataset_doc = serialise.from_doc(dataset.metadata_doc, skip_validation=True)
+        dataset_location = dataset_doc.locations[0] if dataset_doc.locations else None
+
+        uri_list = {
+            name: urljoin(dataset_location, m.path)
+            for name, m in dataset_doc.measurements.items()
```

Comment:
We probably want to include `dataset_doc.accessories.items()` too. 

It includes non-odc metadata files, thumbnails, etc. Basically any files included in the dataset that aren't measurements.

Reply:
Done